content/account/dashboard/console.textile

```diff
@@ -0,0 +1,80 @@
+---
+title: Dev console
+meta_description: "Gain realtime insights into application-wide events, such as connection status changes, channel activity, and event logs.”
+meta_keywords: “Ably dev console, realtime monitoring, connection status changes, channel activity, event logs"
+---
+
+The dev console tab provides realtime insights into application-wide events, such as connection status changes and channel activity. These features enable you to:
+
+* Observe changes in connection status across the application.
+* View activity on specific channels, including message traffic and channel events.
+* Examine event logs to troubleshoot your application.
+
+h2. Application-wide events interface
+
+The application-wide events interface allows you to monitor your application's health and activity in realtime.
+
+The following explains the realtime monitoring tools in the application-wide events interface:
+
+|_. Field |_. Description |
+| *API key* | The API key associated with your Ably application with an additional filter. |
+| *Average application-wide events per second (p/s)* | This metric shows the average number of events occurring per second across your application. For example, if the current rate is 0, no active events are being processed. |
+| *Event log table* | The event log table displays a detailed record of events related to the current client's connection status. This table is essential for diagnosing issues and understanding the flow of events in your application. |
+
+h2. Channels
+
+This following provides step-by-step instructions for connecting to a channel, publishing messages, managing presence, viewing channel and presence history, interacting with the Ably channel via a command-line tool, and controlling channel operations.
```

Comment:
I think the cURL commands should be treated separately, as they aren't really something related to the dev console - i.e. the history parts here aren't really covered in the console itself.